modules/core/src/main/scala/doobieroll/TableColumns.scala

```diff
@@ -104,6 +106,13 @@ sealed abstract case class TableColumns[T](
   def prefixedStr(prefix: String): String =
     allColumns.map(field => s"$prefix.$field").toList.mkString(",")
 
+  /** Return the column name associated with the provided field */
+  def fromFieldF(field: String): Either[NoSuchField, Fragment] =
+    fromFieldStr(field).map(Fragment.const0(_))
```

Comment:
Should we use const instead of const0 to ensure a space is inserted?

Reply:
FYI I'll be removing the F suffix from methods returning Fragments in 0.2.0. Happy to leave it there - I'll remove them all in one go.